infrastructure/cdn-in-a-box/docker-compose.readiness.yml

```diff
@@ -0,0 +1,51 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# This docker-compose file starts a readiness service that will exit
+# successfully when it is able to successfully curl all delivery
+# service example URLs.
+#
+# For example:
+#
+#     docker-compose -f docker-compose.yml -f docker-compose.readiness.yml up
+#
+# Or, start up the main services in the background and run the readiness
+# service in the foreground:
+#
+#     docker-compose up -d
+#     docker-compose -f docker-compose.readiness.yml up
+
+---
+version: '2.1'
+
+services:
+  readiness:
+    build:
+      context: ../..
+      dockerfile: infrastructure/cdn-in-a-box/readiness/Dockerfile
+    env_file:
+      - variables.env
+    hostname: readiness
+    domainname: infra.ciab.test
+    volumes:
+      - shared:/shared
```

Comment:
Does it need the shared volume? Maybe for certs? I'm not sure, just wondering if you know.

Reply:
I believe it's necessary for `insert-self-into-dns.sh`

Reply:
In that case this is answered above.